Fix some favicon issues

felixfontein opened the pull request:
See discussion in #ansible-community.

felixfontein commented:
Ok, I also resolved the last issue: that the icons weren't shown on Safari. The problem is that Safari does **not** support SVGs for icons :-( So if we use SVGs, Safari will not display them. That's why I converted the SVGs to PNGs. I included instructions on how to do the conversion with ImageMagick in the commit message; where should I properly document it? In the README, or somewhere else?

gundalow commented:
Thanks for looking into this.
I guess document in the commit message and in this PRs description.

gundalow commented:
Actually, we have original logos here https://www.ansible.com/logos

felixfontein commented:
Thanks, I used the images from there and added instructions to the README how they are obtained and how I modified them (adjusting the SVG bounding box; rescaling the PNG versions).

gundalow commented on README.rst:
```diff
@@ -4,3 +4,16 @@ sphinx_ansible_theme
 
 Demo-site at https://sphinx-ansible-theme.readthedocs.io/en/latest/
+
+Logos in ``sphinx_ansible_theme/static/images/``
```
Great info, thanks for documenting this

gundalow reviewed (approved):
Thanks for digging into this.

felixfontein commented:
CC @webknjaz

felixfontein commented:
@gundalow @ssbarnea thanks for reviewing and merging!
